_includes/footer.html

```diff
@@ -8,9 +8,9 @@
                       {% if nav.url %}
                            <li><a href="{{site.baseurl}}{{nav.url}}" target="_blank">{{nav.title}}</a></li>
                       {% else %}
-                        {% for ppage in site.pages %}
-                           {% if nav contains ppage.title %}
-                               <li><a href="{{site.baseurl}}{{page.url}}">{{ppage.title}}</a></li>
```

Comment:
here I was using `page` instead of `ppage`. I change `ppage` for `target_page` to be more explicit.